Update quick-create-cli.md

csiebler opened the pull request:
Fixing bash command to actually sleep inside the bash process, otherwise the `sleep 90s` statement is not executed

azurecla commented:
Hi __@lodipalm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (lodipalm). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

lodipalm commented:
Will complete this in -pr

csiebler committed:
Update quick-create-cli.md

Fixing bash command to actually sleep inside the bash process, otherwise the `sleep 90s` statement is not executed
